src/mlpack/core/cv/metrics/r2_score_impl.hpp

```diff
@@ -21,14 +21,8 @@ double R2Score<AdjustedR2>::Evaluate(MLAlgorithm& model,
                                      const DataType& data,
                                      const ResponsesType& responses)
 {
-  if (data.n_cols != responses.n_cols)
-  {
-    std::ostringstream oss;
-    oss << "R2Score::Evaluate(): number of points (" << data.n_cols << ") "
-        << "does not match number of responses (" << responses.n_cols << ")!"
-        << std::endl;
-    throw std::invalid_argument(oss.str());
-  }
+  util::CheckSameSizes(data,(size_t) responses.n_cols, "R2Score::Evaluate()",
+      "responses");
```

Comment:
```suggestion
  util::CheckSameSizes(data, (size_t) responses.n_cols, "R2Score::Evaluate()",
      "responses");
```
Just a simple style fix. :)